aiortc/codecs/h264.py

```diff
@@ -0,0 +1,127 @@
+from itertools import tee
+from struct import unpack_from
+
+from av.packet import Packet
+from av.video.frame import VideoFrame
+from av.codec.context import CodecContext
+
+from ..contrib.media import frame_from_bgr, frame_to_bgr
```

Comment:
The initial intention of contrib.* was that it not be required, as it pulls in opencv. However, I did add opencv to setup.py so I'm not too sure what to do here.

Reply:
This is not required, the avlib already return frames in correct format (yuv420p), I just need figure out how extract the bytes direct

Reply:
Perfect, much better if the library does it for us.